exercises/list-ops/list_ops_test.rb

```diff
@@ -0,0 +1,107 @@
+gem 'minitest', '>= 5.0.0'
+require 'minitest/autorun'
+require_relative 'list_ops'
+
+class ListOpsTest < Minitest::Test
+  def test_count_empty
+    assert_equal 0, ListOps.arrays([])
+  end
+
+  def test_count_normal
```

Comment:
I think `test_count_normal` is not expressive enough in this situation, what is `normal` in this context?  Should we call it `test_count_elements`?